packages/common-server/src/StateValidator.ts

```diff
@@ -22,8 +23,9 @@ export class StateValidator {
         const rootNote = (await engine.findNotes({ fname: "root", vault }))[0];
         try {
           if (rootNote) {
+            const engineNotes = await engine.findNotes({ excludeStub: true });
```

Comment:
We can probably remove this `StateValidator` util once we're on V3? Since there's just a single state.

Reply:
It'll still be useful for validating the metadata (making sure children and parent props are correct compared to the filesystem)